README.md

```diff
@@ -96,14 +97,17 @@ folders for you during the build.
 
 ## Alternatives
 
-As `Cabal` does not currently provide the feature of automatic modules
-discovery, there are some workarounds for this process. One of them is this
-library — `autopack` that uses `Cabal`'s Setup feature to discover all modules
-during the pre-build stage of `Cabal`. Another tool that provides it
-out-of-the-box is [`hpack`](https://github.com/sol/hpack). It is the wrapper on
+* [`hpack`](https://github.com/sol/hpack): wrapper on
 `Cabal` syntax via `YAML` where there is no need to write modules explicitly.
 They are added to the `exposed-modules` field of the `.cabal` file during the
 YAML to `.cabal` step of the tool work process.
+* [`cabal-fmt`](https://oleg.fi/gists/posts/2019-08-11-cabal-fmt.html): a cabal file formatter than can optionally expand source directories during formatting
+
+## Known limitations
+
+Currently, does not work with [`sdist` command](https://github.com/kowainik/autopack/issues/2), because discovered modules are not added to the cabal file.
+If you need to upload your package to hackage, consider using `cabal-fmt`
+before creating an sdist for uploading to generate a valid .cabal file.
```

Comment:
I strongly against giving personal preferences over some tools in this package. Ironically, I never used `cabal-fmt`, so it seems too strange to advocate for this tool here.

I insist on staying unopinionated in my work, which I am trying to be cautious about.
So only neutral description could be accepted, especially when there is more than one way to do things (you could suggest using `hpack` as well as just using clean explicit `cabal` file lists).

Removing this will no harm as you already added information about the tool in the `Alternatives` section.
```suggestion
```